examples/gatsby/gatsby-ssr.js

```diff
@@ -5,7 +5,7 @@ import { renderToString } from 'react-dom/server';
 import { JssProvider } from 'react-jss';
 import getPageContext from './src/getPageContext';
 
-exports.replaceRenderer = ({ bodyComponent, replaceBodyHTMLString, setHeadComponents }) => {
+export const replaceRenderer = ({ bodyComponent, replaceBodyHTMLString, setHeadComponents }) => {
```

Comment:
This is not what I see in the docs: https://www.gatsbyjs.org/docs/ssr-apis/. Is this for v2?

Reply:
yup, I could not get ssr to work properly for few hours, then with some random console logs I've noticed that it doesn't work at all so I tried to export it as const and it worked.

here's repo with some stuff so u can test: https://github.com/blukai/puuffin, try to change `export const replaceRenderer` to `exports.replaceRenderer` and build it (`yarn build && http-server public`), then go to `/contact` page and u'll notice (hit F5), that some styles are broken.

if everything is ok, should I disable eslint rule for this this line?

Reply:
https://github.com/blukai/puuffin/blob/master/package.json#L19

Reply:
We are waiting on gatsby to release a stable version to document it: https://github.com/mui-org/material-ui/issues/12320. I'm putting this PR on hold. Thanks for starting it!

Reply:
How did you get the console logs? When I write console.log() inside gatsby-ssr.js, it does not appear on the screen.

Reply:
@mattes3 yeah, it doesn't work in `gatsby-ssr.js`, I was writing console logs in `getPageContext.js` and `withRoot.js`